### PR TITLE
Add Boxy SVG library to third-party extensions section

### PR DESCRIPTION
**Pull request by LitoMore**
I leave the license field empty since it's a platform app, not a coding library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,7 @@ echo file_get_contents('path/to/package/icons/simpleicons.svg');
...
 | [Blender add-on](https://github.com/mondeja/simple-icons-blender) <img src="https://cdn.simpleicons.org/blender/000/fff" alt="Blender" align=left width=24 height=24> | [@mondeja](https://github.com/mondeja) | ![](https://img.shields.io/github/license/mondeja/simple-icons-blender) | ![](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2Fmondeja%2Fsimple-icons-blender%2Fdevelop%2Fpackage.json&query=%24..%5B'simple-icons'%5D&logo=simpleicons&label=version) |
+| [Boxy SVG library](https://boxy-svg.com/ideas/298/simple-icons-library-provider) <img src="https://cdn.simpleicons.org/boxysvg/000/fff" alt="Boxy SVG" align=left width=24 height=24> | [@Jarek](https://boxy-svg.com/profiles/0000000000/jarek) | | ![](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2Fsimple-icons%2Fsimple-icons%2Fmaster%2Fpackage.json&query=%24.version&logo=simpleicons&label=version) |
```
Comment (adamrusted) [inline README.md:158]: Given, by their own admission, the update script is currently run manually - I don't think it's correct to link to our JSON for the version number. If they automate the releases, sure - but while it's manual I'd prefer we link to the actual version they're using.

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LitoMore) [reply README.md:158]: Since it's open-source, you can know what version they're using. Would you mind leaving it empty?

Comment (adamrusted) [reply README.md:158]: Ideally, we should ask them to publish which version they're using _somewhere_ (API, homepage, etc) that we can read from.

Comment (adamrusted) [inline README.md:158]:
```suggestion
| [Boxy SVG library](https://boxy-svg.com/ideas/298/simple-icons-library-provider) <img src="https://cdn.simpleicons.org/boxysvg/000/fff" alt="Boxy SVG" align=left width=24 height=24> | [@Jarek](https://boxy-svg.com/profiles/0000000000/jarek) | | |
```

**Pull request merged**